app-navigation/navigation.md

```diff
@@ -15,11 +15,15 @@ Every part of the application was designed to give you enough space for showing
 
 On the Action Bar, the following options are available:
 
-* ![](../.gitbook/assets/projectsbutton.png) **Projects** Button - Create a new project, switch between projects.
-* ![](../.gitbook/assets/startserver.png) **Start Server** Button - Starts or stops the background server for mocking requests.
-* ![](../.gitbook/assets/editicon.png) **Edit** Icon - Edit the project name.
-* ![](../.gitbook/assets/addicon.png) **Plus\(+\)** Button - Add an endpoint to the project.
-* ![](../.gitbook/assets/menuicon.png) **Menu** Button - Projects settings, delete all endpoints, delete project.
+- <img align="center" src="../Images/projectsbutton.png" width="93"> **Projects** Button - Create a new project, switch between projects.
+
+- <img align="center" src="../Images/startserver.png" width="93"> **Start Server** Button - Starts or stops the background server for mocking requests.
+
```

Comment:
I think we should use `Start` and `stop` here to be consistent with the other items in the list